packages/RNTester/js/examples/Button/ButtonExample.js

```diff
@@ -26,92 +25,133 @@ exports.description = 'Simple React Native button component.';
 
 exports.examples = [
   {
-    title: 'Simple Button',
-    description: ('The title and onPress handler are required. It is ' +
-      'recommended to set accessibilityLabel to help make your app usable by ' +
-      'everyone.': string),
+    title: 'Button with default styling',
+    description: ('Note: Button with default styling with title="Submit Application"' +
+      'and onPress handler to be called when user taps on button': string),
     render: function (): React.Node {
       return (
-        <RNTesterThemeContext.Consumer>
-          {(theme) => {
-            return (
-              <Button
-                onPress={() => onButtonPress('Simple')}
-                testID="simple_button"
-                color={theme.LinkColor}
-                title="Press Me"
-                accessibilityLabel="See an informative alert"
-              />
-            );
-          }}
-        </RNTesterThemeContext.Consumer>
+        <Button
+          onPress={() => onButtonPress('submitted')}
+          testID="button_default_styling"
+          title="Submit Application"
+          accessibilityLabel="Press to submit your application!"
```

Comment:
The remaining files have a single quote styling in strings, could we have it here also?

Reply:
@YashKumarVerma done